src/Employer/Employer.Web/Orchestrators/PreviewOrchestrator.cs

```diff
@@ -32,33 +33,38 @@ public PreviewOrchestrator(IVacancyClient client, IGeocodeImageService mapServic
 
             var vm = new PreviewVacancyViewModel
             {
-                ApplicationInstructions = string.Empty,
+                ApplicationInstructions = vacancy.ApplicationInstructions,
+                ApplicationUrl = vacancy.ApplicationUrl,
                 CanDelete = vacancy.CanDelete,
                 CanSubmit = vacancy.CanSubmit,
-                ContactName = string.Empty,
-                ContactEmail = string.Empty,
-                ContactTelephone = string.Empty,
-                ClosingDate = vacancy.ClosingDate.Value,
-                Description = string.Empty,
+                ContactName = vacancy.EmployerContactName,
+                ContactEmail = vacancy.EmployerContactEmail,
+                ContactTelephone = vacancy.EmployerContactPhone,
+                ClosingDate = vacancy.ClosingDate.Value.ToString("dd MMM yyyy"),
```

Comment:
I don't think we should assume ClosingDate and PossibleStartDate will not be null. I think this page should render even if elements are missing